integration_tests/help_test.sh

```diff
@@ -4,6 +4,7 @@ set -o pipefail -o errexit -o nounset
 HOME="$TEST_TMPDIR"
 ASPECT="$TEST_SRCDIR/build_aspect_cli/cmd/aspect/aspect_/aspect"
 export HOME
+touch WORKSPACE
```

Comment:
intentional?

Reply:
Yes. Need a WORKSPACE file there so that when the bazel server starts it doesnt throw an error. Bazel starts to generate the available flags